### PR TITLE
Api-Get all payment response

### PR DESCRIPTION
**Pull request by amcspay**

### CODE REVIEW

File: swaggerpay/payswaggerlatest.json [hunk 1]
```diff
@@ -1121,6 +1200,89 @@
         },
         "additionalProperties": false
       },
+      "AMCSPayGetAllPaymentResponseDetails": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
```
Comment (jessahdaculan) [inline swaggerpay/payswaggerlatest.json:1209]: I think we can remove the double quote ("") for this type

File: swaggerpay/payswaggerlatest.json [hunk 2]
```diff
@@ -1121,6 +1200,89 @@
...
+          },
+          "currency": {
+            "type": "string",
+            "nullable": true,
+            "example": "EUR"
+          },
+          "amount": {
+            "type": "Decimal",
+            "nullable": true,
+            "example": "10.02"
```
Comment (jessahdaculan) [inline swaggerpay/payswaggerlatest.json:1229]: I think we can remove the double quote ("") for this type

File: swaggerpay/payswaggerlatest.json [hunk 3]
```diff
@@ -1121,6 +1200,89 @@
...
+          },
+          "paymentType": {
+            "type": "string",
+            "example": "Credit Card",
+            "nullable": false
+          },
+          "paymentTransactionId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
```
Comment (jessahdaculan) [inline swaggerpay/payswaggerlatest.json:1239]: I think we can remove the double quote ("") for this type

File: swaggerpay/payswaggerlatest.json [hunk 4]
```diff
@@ -1121,6 +1200,89 @@
...
+          },
+          "transactionStatus": {
+            "type": "string",
+            "example": "Success",
+            "nullable": false
+          },
+          "paymentId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
```
Comment (jessahdaculan) [inline swaggerpay/payswaggerlatest.json:1254]: I think we can remove the double quote ("") for this type

**Review by jessahdaculan — APPROVED** [review]

**Review by AeraEspinosa — APPROVED** [review]

**Review by DaveMcAMCS — APPROVED** [review]

File: swaggerpay/payswaggerlatest.json (hunk 4 above)
Comment (amcspay) [reply swaggerpay/payswaggerlatest.json]: done

File: swaggerpay/payswaggerlatest.json (hunk 2 above)
Comment (amcspay) [reply swaggerpay/payswaggerlatest.json]: done

**Review by prakashalamanda1984 — COMMENTED** [review]
Looks good

**Pull request merged**